.travis.yml

```diff
@@ -4,4 +4,5 @@ before_script:
     - bundle install
 script:
     - xcodebuild -project Aardvark.xcodeproj -scheme Aardvark -sdk iphonesimulator -configuration Debug -PBXBuildsContinueAfterErrors=0 ACTIVE_ARCH_ONLY=0 build test
+    - xcodebuild -project Aardvark.xcodeproj -scheme "Aardvark-iOS" -sdk iphonesimulator -configuration Release -PBXBuildsContinueAfterErrors=0 ACTIVE_ARCH_ONLY=0 build
```

Comment:
We should get this working in Debug. It's currently failing due to circular imports. I think we'll likely need to:
• Resurrect `ARKLogTypes.h` from the previous PR.
• `#import <Aardvark/ARKLogTypes.h>` before `OBJC_EXTERN void ARKLog`
• `#import <Aardvark/ARKLogTypes.h>` where we currently `#import <Aardvark/Aardvark.h>` from our `.m` files.

Let me know what you think!